wordpress-mobile/WordPress-Android
# Reader Improvements: Recommended site - add tracking

Task #12834
Parent PR #13162

This PR adds tracking for recommended blogs.

To test

- Launch app and go to Reader - Discover tab
- Scroll to recommended blog section
- Click a recommended blog
- Notice following event is tracked:
`🔵 Tracked: reader_suggested_site_visited, Properties: {"blog_id":<blog id>}`
- Go back to the discover feed
- Toggle follow button
- Notice that following event is tracked with correct follow action:
`🔵 Tracked: reader_suggested_site_toggle_follow, Properties: {"blog_id":<blog id>,"follow":true/ false}`

PR submission checklist:
